python/pacemaker/_cts/watcher.py

```diff
@@ -240,23 +239,6 @@ def async_complete(self, pid, returncode, out, err):
             else:
                 messages.append(line)
 
-        if self.limit and not found_cursor:
-            self._hit_limit = True
-            self.debug("Got %d lines but no cursor: %s" % (len(out), self.offset))
```

Comment:
We re-add a "did we ever find a cursor" check in a later commit. However, it's implemented differently and throws an exception if no cursor, and none of the dropped code comes back.